site/docs/api/sources-sinks.md

```diff
@@ -1035,11 +1035,12 @@ doesn't use XA transactions.
 
 Change Data Capture (CDC) refers to the process of observing changes
 made to a database and extracting them in a form usable by other
-systems, for the purposes of replication, analysis and many many more.
+systems, for the purposes of replication, analysis and many more.
 
 Change Data Capture is especially important to Jet, because it allows
-for the _integration with legacy systems_. Database changes form a
-stream of events which can be efficiently processed by Jet.
+for the _streaming of changes from databases_, which can be efficiently
+processed by Jet. One consequence of this is also simplified integration
```

Comment:
We don't need to mention legacy systems at all. It can instead say simplify integration with relational databases or something like help move from relational databases to an event-driven architecture or similar. Have a look at debezium.io and read their messaging, there isn't anything about legacy systems.